README.md

```diff
@@ -140,6 +140,12 @@ Full variables list could be found [here](https://github.com/lidofinance/lido-or
 > docker run --env EXECUTION_CLIENT_URI={value} --env CONSENSUS_CLIENT_URI={value} --env KEYS_API_URI={value} --env LIDO_LOCATOR_ADDRESS={value} lidofinance/oracle:{tag} {type}
 > ```
 
+## Manual mode
+
+Oracle could be executed once in "manual" mode. To do this setup `DAEMON` variable to 'True'.
```

Comment:
```suggestion
Oracle could be executed once in "manual" mode. To do this setup `DAEMON` variable to 'False'.
```

And I think it should be a more verbose paragraph about why we need this

Reply:
Done

Reply:
`docker run -ti --env-file .env --rm lidofinance/oracle:{tag} {type}` add command example